core/src/datayoga_core/jmespath_custom_functions.py

```diff
@@ -142,3 +142,10 @@ def _func_in(self, element: Any, iterable: Iterable) -> bool:
         Returns True if the iterable contains the given element.
         """
         return element in iterable
+
+    @functions.signature({"types": ["string"]})
+    def _func_json_parse(self, data: str) -> Any:
+        """\
```

Comment:
Either remove \ or put the comment in one line.

Reply:
✅